modules/client-configuration/pages/client-upgrades-sp-migration.adoc

```diff
@@ -1,7 +1,7 @@
 [[client-upgrades-spmigration]]
-= Service Pack Migration
```

Comment:
If it does not mean to much trouble. Can we also rename the file to something like `client-upgrated-product-migration`